kernel/trunk/boot/bootcode.inc

```diff
@@ -894,15 +894,15 @@ else if lang eq et
         mov     byte [time_str+10], ' '
 @@:
 else if lang eq sp
-; esperar 5/4/3/2 segundos, 1 segundo
+; esperar 5/4/3/2 segundoS, 1 segundo
```

Comment:
What is `segundoS`?

Reply:
It shows the meaning of code below. In Spanish you add Ssuffix for plural. So I left this for clarity